sim/rogue/mutilate.go

```diff
@@ -84,8 +87,12 @@ func (rogue *Rogue) registerMutilateSpell() {
 			result := spell.CalcOutcome(sim, target, spell.OutcomeMeleeSpecialHit) // Miss/Dodge/Parry/Hit
 			if result.Landed() {
 				rogue.AddComboPoints(sim, 2, spell.ComboPointMetrics())
-				rogue.MutilateOH.Cast(sim, target)
+
+				if rogue.HasRune(proto.RogueRune_RuneWaylay) {
```

Comment:
Disable Waylay in the same way as Ambush and Backstab due to not applying to bosses due to slow effect